.github/workflows/rust.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       run: rustup update stable
     - name: Build
       run: cargo build --verbose
+    - name: Build no-std
+      run: cargo build --no-default-features --verbose
```

Comment:
I think we will also want to run the tests in a `no-std` mode

Reply:
this can be another PR but for reference this is the latest version of this CI suite I like for this set of repos:

https://github.com/ralexstokes/ethereum-consensus/blob/main/.github/workflows/rust.yml

Reply:
I updated the CI. https://github.com/ralexstokes/ssz-rs/pull/25/commits/f4cc01baf8fa43578325d49adf7cc71258b8c208

Running tests in no-std mode isn't something that really makes sense because that would mean having to remove the crates used in testing, like the `hex` crate. As far as I know running tests can still depend on std (our bridge test code also relies on std).